rotkehlchen/tests/unit/decoders/test_curve.py

```diff
@@ -2463,3 +2463,60 @@ def test_liquidity_withdrawal(database, gnosis_inquirer, gnosis_accounts, load_g
         ),
     ]
     assert events == expected_events
+
+
+@pytest.mark.vcr(filter_query_parameters=['apikey'])
+@pytest.mark.parametrize('load_global_caches', [[CPT_CURVE]])
+@pytest.mark.parametrize('gnosis_accounts', [['0xc37b40ABdB939635068d3c5f13E7faF686F03B65']])
+def test_monerium_eure(gnosis_inquirer, gnosis_accounts, load_global_caches):
+    """Regression test for https://github.com/rotki/rotki/issues/8452"""
```

Comment:
Add a tiny bit more details of what you are testing here. I guess you test that the events of v1 do not appear?